pkg/controller/istio/istio_controller.go

```diff
@@ -406,6 +410,40 @@ func (r *ReconcileConfig) deleteRemoteIstios(config *istiov1beta1.Istio, logger
 	}
 }
 
+func (r *ReconcileConfig) checkIstioCount(ctx context.Context, logger logr.Logger, namespace string, config *istiov1beta1.Istio) (reconcile.Result, error) {
+	count, err := r.getIstioCount(ctx, namespace)
+	if err != nil {
+		logger.Error(err, "cannot get Istio CR count")
+		return reconcile.Result{
+			Requeue:      true,
+			RequeueAfter: 30 * time.Second,
+		}, nil
+	}
+	logger.Info("Got Istio CR count", "count", count)
```

Comment:
Not sure I'd log this. Maybe debug level log?

Reply:
Leftover from debugging, thanks!